rkt/run.go

```diff
@@ -93,6 +95,38 @@ func init() {
 	cmdRun.Flags().SetInterspersed(false)
 }
 
+func getStage1Hash(s *store.Store, stage1ImagePath string) (*types.Hash, error) {
+	fn := &finder{
+		imageActionData: imageActionData{
+			s: s,
+		},
+		local:    true,
+		withDeps: false,
+	}
+	isDefault := stage1ImagePath == defaultStage1Image
+
+	var (
+		s1img *types.Hash
+		err   error
+	)
+	// with a default stage1, first try to get the image from the store
+	if isDefault {
+		stage1AppName := fmt.Sprintf("coreos.com/rkt/stage1:%s", url.QueryEscape(version.Version))
```

Comment:
Yeah I am just wondering if we can consolidate the two to ensure they're in sync